src/components/pages/ParentDash/components/Nav.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../dash.css';
+
+import { Layout, Menu } from 'antd';
+
+// Nav bar for parent dash
+const Nav = () => {
```

Comment:
Should this Nav be named in a way to distinguish it from the 'child nav'? Not sure how it's being implemented, but I am curious since it is commented as 'Nav for parents'

Reply:
Good point, I can rename